#! /bin/sh /usr/share/dpatch/dpatch-run
## 08_gcc-4.3_fixes.dpatch by Cyril Brulebois <cyril.brulebois@enst-bretagne.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix gcc-4.3 FTBFS, patch by KiBi (Closes: #455304).

--- a/examples/cpp/encode/file/main.cpp
+++ b/examples/cpp/encode/file/main.cpp
@@ -33,6 +33,8 @@
 #include "FLAC++/metadata.h"
 #include "FLAC++/encoder.h"
 
+#include <cstring>
+
 class OurEncoder: public FLAC::Encoder::File {
 public:
 	OurEncoder(): FLAC::Encoder::File() { }
